src/android/SplashScreen.java

```diff
@@ -393,6 +390,7 @@ public void onCancel(DialogInterface dialog) {
 
                 spinnerDialog.getWindow().clearFlags(WindowManager.LayoutParams.FLAG_DIM_BEHIND);
                 spinnerDialog.getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
+                spinnerDialog.getWindow().setGravity(Gravity.CENTER);
```

Comment:
What does this have to with the image? Again separate PR is recommended for different target changes.

Reply:
This [PR](https://github.com/apache/cordova-plugin-splashscreen/pull/234) is already about this.